src/config/index.js

```diff
@@ -0,0 +1,12 @@
+const config = {
```

Comment:
This is beautiful, have some really cool ideas for when it comes to stitching everything together via smoothie!

Reply:
Can't wait for that sweet smoothie